tools/permuter_settings.toml

```diff
@@ -2,7 +2,7 @@
 "g[DS]P.*" = "void"
 "gDma.*" = "void"
 "G_IM_SIZ_.*" = "int"
-"G_[AC]CMUX.*" = "int"
+"G_[AC]C.*" = "int"
```

Comment:
Yeah that's the intention behind the change, I needed to make the permuter preserve the CC_ macros as well while working on the file, otherwise it just would just crash on them.